go.sum

```diff
@@ -18,6 +18,8 @@ go.uber.org/automaxprocs v1.5.3 h1:kWazyxZUrS3Gs4qUpbwo5kEIMGe/DAvi5Z4tl2NW4j8=
 go.uber.org/automaxprocs v1.5.3/go.mod h1:eRbA25aqJrxAbsLO0xy5jVwPt7FQnRgjW+efnwa1WM0=
 golang.org/x/crypto v0.16.0 h1:mMMrFzRSCF0GvB7Ne27XVtVAaXLrPmgPC7/v0tkwHaY=
 golang.org/x/crypto v0.16.0/go.mod h1:gCAAfMLgwOJRpTjQ2zCCt2OcSfYMTeZVSRtQlPC7Nq4=
+golang.org/x/exp v0.0.0-20231206192017-f3f8817b8deb h1:c0vyKkb6yr3KR7jEfJaOSv4lG7xPkbN6r52aJz1d8a8=
```

Comment:
This was added here as `exp/slices` is used in test utility function. That means it will not be part of the binary, but as we want to avoid adding dependencies in general, it would be nice to not pull it in for that single reason.

Reply:
The `slices` package has been included in the standard library with Go 1.21. I had to include it from `x/exp` so that this code compiles with Go 1.20. If you see a problem with adding this dependency I can certainly find a way to write the test code without it.

Reply:
Yes please, trying to keep dependencies to a minimum here.

Reply:
Pushed a version that is not using `slices` any longer.